### PR TITLE
SSCS-7299: IDAM - Review

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/CitizenController.java [hunk 1]
```diff
@@ -61,10 +61,6 @@ public CitizenController(CitizenLoginService citizenLoginService,
             @ApiParam(value = "user authorisation header", example = "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdW") @RequestHeader(AUTHORIZATION) String authorisation,
             @ApiParam(value = "tya number for an user and appeal", example = "A123-B123-c123-Dgdg") @PathVariable("tya") String tya) {
 
-        if (!idamService.verifyTokenSignature(authorisation)) {
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/CitizenController.java:64]: It's ok to remove these lines?

Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/CitizenController.java:64]: Yes now we have spring-security which authorised based on token

**Review by jackmaloney — DISMISSED** [review]

**Comment by srochani** [comment]
Closed the PR And reopend

**Comment by srochani** [comment]
Closing and opening new PR

**Pull request closed without merging**